scripts/install-relayer.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+RESET_CONFIG="${RESET_CONFIG:-0}"
+VERSION="${VERSION:-v2.0.0}"
+RELAYER_PATH=$HOME/.relayer
+RELAYER_CONFIG=$RELAYER_PATH/config/config.yaml
+PROVENANCE_CONFIG=scripts/relayer-config.yaml
+
+# Install relayer if it doesn't exist
+if ! [ -x "$(command -v rly)" ]; then
+    echo "relayer not installed. Installing..."
+    # Install relayer
+    DIR=$(pwd)
+    git clone https://github.com/cosmos/relayer.git /tmp/relayer
+    cd /tmp/relayer
```

Comment:
if the checkout/directory creation may fail ...  add `|| exit 1 ` to halt the script with an error here.